app/decorators/submission_decorator.rb

```diff
@@ -33,4 +28,22 @@ def doi
   def label
     raise Exception.new('Should be extended by subclasses')
   end
+
+  def bip_link
```

Comment:
It returns url, not link (like e.g. `link_to`), so `bip_url` (or `data_tables_url`, IMO better).